ui/src/app/modules/projects/modules/integrations/components/integration/integration.component.html

```diff
@@ -1,7 +1,23 @@
 <div class="integration-screen">
-
-    <div class="integration-tab">
-        <h1>Integrations</h1>
+    <div class="steps">
+        <h1><strong> Step 1 :</strong> Create a <strong> Project Logs </strong> project</h1>
+        <mat-card>
+            <p>
+                Before you can add <strong> Project Logs </strong> to your app, you need to create a new <strong> Project Logs </strong> project to connect to your app. Please in the left panel click on New Project.
+            </p>
+        </mat-card>
+    </div>
+    <div class="steps">
+        <h1><strong>Step 2 :</strong> Copy AppKey</h1>
+        <mat-card>
+            <p>
+                To connect your app with <strong> Project Logs </strong>, you need to copy the App key. Please in
```

Comment:
After you create the project you will need to get the api key in order to develop your integrations.